tldextract/tldextract.py

```diff
@@ -229,7 +226,7 @@ def _get_tld_extractor(self):
         if self._extractor:
             return self._extractor
 
-        tlds = frozenset()
```

Comment:
You are right! Since we changed the way JSON is being encoded/decoded we can use frozenset again.